simtools/corsika/corsika_runner.py

```diff
@@ -329,12 +331,12 @@ def get_file_name(self, file_type, **kwargs):
             f"_{kwargs['label']}" if "label" in kwargs and kwargs["label"] is not None else ""
         )
         file_name = (
-            f"corsika_run{kwargs['run']}_{kwargs['primary']}_"
+            f"corsika_run{kwargs['run']:06}_{kwargs['primary']}_"
```

Comment:
Do I understand it correctly that this is due to the 6-digit run numbers in corsika and you want to make sure that this is enforced?

Reply:
OK - but it defines a max range of run numbers, or?

And people should use `sort -n` to solve above issue.

Reply:
It doesn't limit the run numbers, they can be larger than 999,999, it's just that after that we won't guarantee the consecutive order.

> And people should use sort -n to solve above issue.

Sure, we could ask people to use that and implement similar "tools" in the various places (when dumping file lists from the grid, listing files in ctapipe, etc.). However, it's probably easier to simply pad the numbers =)